rebasehelper/application.py

```diff
@@ -716,8 +757,15 @@ def run(self):
         if not self.conf.keep_workspace:
             self._delete_workspace_dir()
 
+        self.generate_patch()
+
         if self.debug_log_file:
             logger.info("Detailed debug log is located in '%s'", self.debug_log_file)
+            logger.info("Modified patches and specfile needed for the rebased version "
+                        "are located in '%s'", self.rebased_sources_dir)
+            logger.info("Patch '%s' containing changes of patches nad specfile between "
```

Comment:
There is a typo, and I think I would just use `rebased spec file and patches`. So `Patch '%s' containing changes between original and rebased spec file and patches ...`, and `Rebased spec file and patches are located ...`.